src/traces/candlestick/attributes.js

```diff
@@ -51,6 +51,9 @@ module.exports = {
     hovertext: OHLCattrs.hovertext,
 
     whiskerwidth: extendFlat({}, boxAttrs.whiskerwidth, { dflt: 0 }),
+    showwhiskers: boxAttrs.showwhiskers,
+    sizemode: boxAttrs.sizemode,
```

Comment:
I don't think candlesticks make sense without whiskers, and certainly `sizemode` doesn't apply here. So we'll need to find another way to make them draw properly without creating these attributes.